### PR TITLE
Implemented ping command with message argument

### PR DESCRIPTION
**Pull request by sazzad16**
1. Added `String ping(String message)` in BasicCommands.
2. Added implementations for `BasicCommands.ping(String message)`
3. Added test codes for `Jedis.ping(String message)`

### CODE REVIEW

File: src/main/java/redis/clients/jedis/BinaryClient.java [hunk 1]
```diff
@@ -112,6 +112,10 @@ public void ping() {
     sendCommand(Command.PING);
   }
 
+  public void ping(final String message) {
```
Comment (marcosnils) [inline src/main/java/redis/clients/jedis/BinaryClient.java]: in Binaryclient arguments usually are []byte instead of strings.

Comment (sazzad16) [inline src/main/java/redis/clients/jedis/BinaryClient.java]: Thanks for the review. Will give an update.

**Comment by marcosnils** [comment]
LGTM!. @HeartSaVioR ?

**Comment by HeartSaVioR** [comment]
LGTM! Thanks @sazzad16 for the contribution.

**Comment by HeartSaVioR** [comment]
Merged into master and 2.10.

**Pull request merged**